lib/net.c

```diff
@@ -116,19 +116,19 @@ ethertype_filter_add(uint8_t port_id, uint16_t ether_type, uint16_t queue_id)
 		RTE_ETH_FILTER_ADD,
 		&filter);
 	if (ret == -ENOTSUP) {
-		RTE_LOG(ERR, PORT,
+		RTE_LOG(NOTICE, PORT,
 			"Hardware doesn't support adding an EtherType filter for 0x%02hx on port %hhu!\n",
 			ether_type, port_id);
 		ret = -1;
 		goto out;
 	} else if (ret == -ENODEV) {
-		RTE_LOG(ERR, PORT,
+		RTE_LOG(NOTICE, PORT,
 			"Port %hhu is invalid for adding an EtherType filter for 0x%02hx!\n",
 			ether_type, port_id);
```

Comment:
This is a mistake from previous patch, we may switch the ``ether_type`` and ``port_id`` here.